### ISSUE TITLE
Devise a strategy and repository structure for sample/starter apps

### ISSUE THREAD

**Issue by msgilligan** [SupernautApp/SupernautFX#12]

**Comment by FDelporte** [comment]
Maybe a bit controversial but maybe the simple getting started example repositories could be both Maven and Gradle? I've done a similar thing in https://github.com/Pi4J/pi4j-example-minimal with some Gradle-assistance as I'm a Maven user ;-) See https://github.com/Pi4J/pi4j-example-minimal/pull/3

**Comment by msgilligan** [comment]
> both Maven and Gradle? I've done a similar thing in https://github.com/Pi4J/pi4j-example-minimal

Yeah. I was thinking that might be the way to go. How is it being received by your users?
